docs/config/README.md

```diff
@@ -366,3 +366,28 @@ The module will be shown if any of the following conditions are met:
 [username]
 disabled = true
 ```
+
+## Command Duration
+
+The `cmd_duration` module shows how long the last command took to execute.
+The module will be shown only if the command took longer than two seconds, or
+the `min_time` config value, if it exists.
+
+NOTE: Command duration is currently **not supported in `bash`**. See
```

Comment:
As we're using Vuepress for our configuration, this may be a good place to put a "custom container: https://vuepress.vuejs.org/guide/markdown.html#custom-containers

I was thinking either a tip or a warning.

```
::: tip NOTE
Command duration is currently **not supported in `bash`**. See
[this issue](https://github.com/starship/starship/issues/124) for more details.
:::
```

Reply:
Great idea. I had initially bolded that text only because I wanted it to be attention-grabbing and bolding the whole thing seemed obnoxious, but a warning works much better for that.